pygmt/src/grdcontour.py

```diff
@@ -95,6 +97,34 @@ def grdcontour(self, grid, **kwargs):
         separator for the two labels instead.
     {perspective}
     {transparency}
+
+
+    Example
+    -------
+    >>> import pygmt
+    >>> # load the 15m grid with "gridline" registration in a specified region
+    >>> grid = pygmt.datasets.load_earth_relief(
+    ...     resolution="15m",
+    ...     region=[-92.5, -82.5, -3, 7],
+    ...     registration="gridline",
+    ... )
```

Comment:
Maybe we should state the unit in long form in the comment?
```suggestion
    >>> # load the 15 arc minute grid with "gridline" registration in a specified region
    >>> grid = pygmt.datasets.load_earth_relief(
    ...     resolution="15m",
    ...     region=[-92.5, -82.5, -3, 7],
    ...     registration="gridline",
    ... )
```